src/test/java/com/karankumar/bookproject/backend/service/BookServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package com.karankumar.bookproject.backend.service;
+
+import com.karankumar.bookproject.backend.entity.Author;
+import com.karankumar.bookproject.backend.entity.Book;
+import com.karankumar.bookproject.backend.entity.PredefinedShelf;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@SpringBootTest
+public class BookServiceTest {
+
+    private static PredefinedShelfService shelfService;
+
+    private static AuthorService authorService;
+    private static BookService goalService;
+
+    private Book bookWithoutShelf;
+    private Book bookWithoutAuthor;
+    private Book validBook;
+    private Author author;
+
+    @BeforeEach
+    public void setup(@Autowired BookService goalService, @Autowired AuthorService authorService,@Autowired PredefinedShelfService shelfService) {
```

Comment:
Could you drop the `PredefinedShelfService` argument on the next line as this line currently exceeds 120 characters?